components/TextFields/src/MDCTextField.m

```diff
@@ -34,6 +34,15 @@ @interface MDCTextField ()
 
 @property(nonatomic, strong) MDCTextInputCommonFundament *fundament;
 
+/** 
+ 
+ This constraint controls the default center Y of the underline view. 
```

Comment:
Delete the leading empty line. Probably just "Constraint for the default center y-coordinate of the underline view."